carbonmark-api/src/app.constants.ts

```diff
@@ -56,3 +56,6 @@ export const TOKEN_ADDRESSES = {
 export const RPC_URLS = {
   polygonTestnetRpc: "https://rpc-mumbai.maticvigil.com",
 };
+
+/** Message shared with frontend, to be combined with user's nonce and signed by private key. */
+export const SIGN_PROFILE_MESSAGE = `${process.env.SIGN_PROFILE_MESSAGE}`;
```

Comment:
Is this different from the current `AUTHENTICATION_MESSAGE` in vercel? 

<img width="1098" alt="image" src="https://github.com/KlimaDAO/klimadao/assets/7104689/0572fb9c-8341-4f0c-8edc-1788466a9fe8">

Reply:
renamed to make it specific for this operation (profile puts/post)